README.md

```diff
@@ -200,13 +200,29 @@ curl -X POST -H "Content-Type: application/json" http://localhost:8080/api/v1/re
 The REST API and the worker will restart if the source code is modified. Please note that the REST
 API may stop restarting if there is a syntax error.
 
+#### Rebuilding Images
+
+If you suspect that the images used for [`docker-compose`](#run-a-containerized-development-environment) are out of date, you can
+run the containerized development environment while forcing a rebuild of the images with the following
+command:
+
+```bash
+make run-start-over
```

Comment:
The target `run-start-over` doesn't exist, right?

Reply:
addressed.